src/components/pages/landing/FractalVerification.js

```diff
@@ -200,13 +203,13 @@ export const ScanFace = ({ setActiveTabIndex }) => {
             status: 'Fractal Approved',
           });
           log_event({
-            event_log: 'User is approved from Fractal',
+            event_log: 'User is approved from Fractal' + JSON.stringify(resp),
           });
           setActiveTabIndex(2);
         }
         if (resp?.error) {
           log_event({
-            event_log: resp?.error,
+            event_log: 'Error received from oracle API' + resp?.error,
```

Comment:
Let's make convention for errors like:`<Entity>: <error title>. <error description>`
So e.g.
```js
"Oracle: API Error. " + resp.error
"Fractal: User is not approved! " + JSON.stringify(resp)
"Fractal: User succesfully approved! " + JSON.stringify(resp)
```

Reply:
makes sense, will update